lib/base-chips/field-types/text.js

```diff
@@ -4,12 +4,10 @@ const escape = require("escape-html");
 
 module.exports = {
 	name: "text",
-	full_text_search_enabled: function(params){
-		if (params && params.include_in_search){
-			return true;
-		} else if (params && params.max_length > 200){
-			return true;
-		} else {
+	has_index: function(params){
```

Comment:
That's the whole deal with this change. `full_text_search` supported only `text` indexes. Now we can support arbitrary indexes, like `sphere2d`. From user-reference:

> has_index: optional. Whether or not to instruct the Datastore to create an index on that field. In order to include the contents of the fields of this type in full-text-search, return "text" here.